src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -21,9 +21,10 @@ storiesOf('Components/Tag', module)
       tag(
         merge(dataLink, {
           tag: {
-            label: text('Label', 'Link tag'),
-            path: text('Url', '/example'),
+            label: text('Label', dataLink.tag.label),
+            path: text('Url', dataLink.tag.path),
```

Comment:
MMhh, but if we are doing this in order to match as much as possible the specs, why we name the knob "url" when the property passed to the template is path? I understand that in any case a bit of ambiguity is in place, considering that both a path and an url can be used, so that's what i would do, we name it: `Path (accepts also a full url)`, what do you think?